pdr_backend/accuracy/readme.md

```diff
@@ -0,0 +1,47 @@
+# pdr_backend/accuracy
+
+This document provides instructions on how to deploy and run the `app.py` script in the `pdr_backend/accuracy` directory.
+
+## Requirements
+
+- Python 3.x
+- Required Python packages (listed in `requirements.txt`)
+
+## Deployment
+
+```bash
+git clone <repository_url>
+cd pdr_backend
+
+pip install -r requirements.txt
+```
+
+## Usage
+
+The `app.py` script is used to calculate the accuracy of a model based on the predicted and true values.
+
+```bash
+python pdr_backend/accuracy/app.py
+```
+
+
+The script uses the `GQLDataFactory` to fetch the predictions. Be sure to provide the correct values for `st_timestr` in the `ppss.yaml` file to get the correct predictions. It needs to be at least `28 days ago` from the current date. 
+    
+The script will output the accuracy of the model based on the provided values to a file named `pdr_backend/accuracy/output/accuracy.json`.
+
+## Flask API
+
+The `app.py` script provides a Flask endpoint with `/statistics` route to get the accuracy of the model.
+
+```bash
+curl http://localhost:5000/statistics
+```
+
+The endpoint will return the accuracy of the model in JSON format.
```

Comment:
We could provide a JSON example of the response with all the fields and also mention that the API returns multiple values not just accuracy(`token_name, total_staked_yesterday, total_staked_today`), hence the statistics names.

Reply:
Ok I will bring an example JSON and mention about it

Reply:
I put the information to the "Usage" subtitle, under the `json` file info